api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -812,4 +812,26 @@ public Order discontinueOrder(Order orderToDiscontinue, String reasonNonCoded, D
 	 */
 	@Authorized({ PrivilegeConstants.EDIT_ORDERS, PrivilegeConstants.ADD_ORDERS })
 	public OrderGroup saveOrderGroup(OrderGroup orderGroup) throws APIException;
+	
+	/**
+	 * Fetches all order groups for the specified patient
+	 * 
+	 * @param patient the patient to match on
+	 * @return list of matching OrderGroup * @since 2.3
```

Comment:
Why are you putting the since annotation on the same line as the return?

Reply:
i have fixed it